lib/et/fallback_connection.rb

```diff
@@ -0,0 +1,29 @@
+require "faraday"
+require "faraday_middleware"
+
+module ET
+  class FallbackConnection
+    def initialize(opts = {}, &block)
+      @connection = Faraday.new(opts, &block)
+
+      @fallback_connection = Faraday.new(opts.merge(:ssl => {:verify => false}))
+    end
+
+    def with_ssl_fallback(&block)
```

Comment:
Not to nitpick, but "with_ssl_fallback" sounds like the connection "falls back" to ssl, instead of starting with it.